src/spec/ng-bootstrap/popover.spec.ts

```diff
@@ -238,6 +238,30 @@ describe('popover', () => {
       fixture.detectChanges();
       expect(getWindow(fixture.nativeElement)).toBeNull();
     });
+
+    it('should properly set aria-describedBy and id for popover', () => {
+      const fixture = createTestComponent(
+        `<div id="div-container" style="margin: 400px" popover="Great tip!" popoverTitle="Title"></div>`
+      );
+      const directive = fixture.debugElement.query(
+        By.directive(PopoverDirective)
+      );
+
+      dispatchMouseEvent(directive.nativeElement, 'click');
+      fixture.detectChanges();
+
+      const windowEl = getWindow(fixture.nativeElement);
+
+      let popoverId = windowEl.getAttribute('id');
```

Comment:
that causes a lint issue
Looks like, you have to use `const` here, because, popoverId was never reassigned somewhere in this file

Reply:
Yep, i see that